templates/default/nginx-proxy-capture-agent-manager.conf.erb

```diff
@@ -0,0 +1,10 @@
+    location / {
+        proxy_buffering off;
+        proxy_pass http://unix:/home/<%= @capture_agent_manager_usr_name %>/sock/<%= @capture_agent_manager %>.sock;
```

Comment:
i copied from an example i googled... but looks nice that it will not allow anyone to access the app via some tcp port. do you think this has drawbacks?